### PR TITLE
Added link prop to inlineCTA component

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: components/base/InlineCTA.tsx [hunk 1]
```diff
@@ -54,17 +58,22 @@ const ArrowStyle = styled.div`
 type Props = {
   icon: string;
   text: string;
+  href: string;
```
Comment (Finholt) [inline components/base/InlineCTA.tsx:61]: Will we ever use this component for external links (that would open in a new tab)? For other link components like the Card, I've been using `link?: { href?: string; external?: boolean };` as a prop to allow for that

It's totally fine if we're not planning on using this particular component like that, but I wanted to make sure we considered it!

Comment (ShreyaThapa) [reply components/base/InlineCTA.tsx:61]: Good question! We have an external CTA component that has a different design (perhaps to distinguish it between external and internal? not sure 🤷 ) but I wouldn't be against accounting for both external and internal linking via the Inline CTA.

Comment (Finholt) [reply components/base/InlineCTA.tsx:61]: Oh that's a good point! I forgot we had another component for that specifically. Thanks for reminding me 🙂

**Review by Finholt — APPROVED** [review]

File: components/base/InlineCTA.tsx (hunk 1 above)
Comment (ShreyaThapa) [reply components/base/InlineCTA.tsx:61]: Thanks for the APPROVED. As an aside, if in the future we do decide to use this CTA for an external link, we can always add that option!

**Pull request merged**